src/ServiceControl.LicenseManagement/ServiceControl.LicenseManagement.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net40</TargetFramework>
```

Comment:
is this the right target framework?

Reply:
Yes, it's used by the installer project which is 40 so this has to match